### ISSUE TITLE
Test Failure JIT.jit64 JIT.Methodical on windows arm64 Checked no_tiered_compilation

### ISSUE THREAD

**Issue by eerhardt** [dotnet/runtime#45440]
CoreCLR Pri0 Runtime Tests Run windows arm64 checked leg failed [here](https://dev.azure.com/dnceng/public/_build/results?buildId=904394&view=logs&j=bb94f37e-a09b-5b51-09b2-ab78e7aa5103&t=506cadc9-3cc4-5d96-6847-011f75a33382&l=37):

```
  Job ef919131-84b9-4295-abf8-752ebc3f13a3 on Windows.10.Arm64v8.Open is completed with 32 finished work items.
  Job 82b5e9d5-72b6-44b7-a645-9cabf5483f92 on Windows.10.Arm64v8.Open is completed with 32 finished work items.
  Stopping Azure Pipelines Test Run CoreCLR windows arm64 Checked @ Windows.10.Arm64v8.Open
  Stopping Azure Pipelines Test Run CoreCLR windows arm64 Checked no_tiered_compilation @ Windows.10.Arm64v8.Open
F:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\6.0.0-beta.20573.2\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(76,5): error : Work item JIT.Methodical in job 82b5e9d5-72b6-44b7-a645-9cabf5483f92 has failed. [F:\workspace\_work\1\s\src\tests\Common\helixpublishwitharcade.proj]
F:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\6.0.0-beta.20573.2\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(76,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/82b5e9d5-72b6-44b7-a645-9cabf5483f92/workitems/JIT.Methodical/console [F:\workspace\_work\1\s\src\tests\Common\helixpublishwitharcade.proj]
##[error].packages\microsoft.dotnet.helix.sdk\6.0.0-beta.20573.2\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(76,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item JIT.Methodical in job 82b5e9d5-72b6-44b7-a645-9cabf5483f92 has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/82b5e9d5-72b6-44b7-a645-9cabf5483f92/workitems/JIT.Methodical/console
F:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\6.0.0-beta.20573.2\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(76,5): error : Work item JIT.jit64 in job 82b5e9d5-72b6-44b7-a645-9cabf5483f92 has failed. [F:\workspace\_work\1\s\src\tests\Common\helixpublishwitharcade.proj]
F:\workspace\_work\1\s\.packages\microsoft.dotnet.helix.sdk\6.0.0-beta.20573.2\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(76,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/82b5e9d5-72b6-44b7-a645-9cabf5483f92/workitems/JIT.jit64/console [F:\workspace\_work\1\s\src\tests\Common\helixpublishwitharcade.proj]
##[error].packages\microsoft.dotnet.helix.sdk\6.0.0-beta.20573.2\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(76,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item JIT.jit64 in job 82b5e9d5-72b6-44b7-a645-9cabf5483f92 has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/82b5e9d5-72b6-44b7-a645-9cabf5483f92/workitems/JIT.jit64/console
```

Unfortunately looking at the logs doesn't tell me why the test failed. The console logs just have:

```
Executed on DDARM64-174
```

**Comment by jkoritzinsky** [comment]
Duplicate of https://github.com/dotnet/core-eng/issues/11539

@MattGal can you take a look?

**Comment by jkoritzinsky** [comment]
This particular machine seems to be having issues.

**Comment by MattGal** [comment]
> Duplicate of [dotnet/core-eng#11539](https://github.com/dotnet/core-eng/issues/11539)
> 
> @MattGal can you take a look?

Sure!  Will take it offline as well.

**Comment by MattGal** [comment]
Added some notes there, I think this is the classic "CoreCLR tests run afoul of antivirus scanners", I'll work with @ilyas1974 to get this unblocked.

**Comment by directhex** [comment]
Theoretically fixed by https://github.com/dotnet/core-eng/issues/11539
